ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -24,10 +24,19 @@ echo $master_type
 
 ENDPOINT="${AZURESTACK_ENDPOINT}"
 echo "ASH ARM Endpoint: ${ENDPOINT}"
+
 cp "/var/run/azurestack-cluster-secrets/service-principal" "${SHARED_DIR}/osServicePrincipal.json"
+if [[ -f "${CLUSTER_PROFILE_DIR}/cloud_name" ]]; then
```

Comment:
To be more safe, suggest to retrieve the value of `cluster_name`, judge it to check if it is equal to `wwt`